src/lib.rs

```diff
@@ -64,8 +74,26 @@ pub fn drop_table(table: &str) -> io::Result<()> {
     Ok(())
 }
 
+#[allow(unused_must_use)]
```

Comment:
This warning doesn't fire anyway.

Types marked `must_use` are generally marked as such for good reasons; I'd like to encourage you not to silence this warning.